### ISSUE TITLE
got error when "yarn upgrade" after micromodal 0.4.3

### ISSUE THREAD

**Issue by cron13** [ghosh/Micromodal#306]
Hi, after upgrade micromodal to 0.4.3 i got error
```
error An unexpected error occurred: "ELOOP: too many symbolic links encountered, stat '/Users/user/Library/Caches/Yarn/v6/npm-micromodal-0.4.3-7c074b3019bf52b860c8d3b2dd9513fe7b2769ea-integrity/node_modules/micromodal/README.md'".
```
after run yarn upgrade in project directory

I use macos 10.15.3, also i got same error on ubuntu 18.04.4

**Comment by ghosh** [comment]
Yes thanks for reporting. We are aware of this issue and working to resolve it.

**Comment by ghosh** [comment]
This is fixed in `0.4.4`.
